src/Compilers/CSharp/Portable/Symbols/Source/SourceEventSymbol.cs

```diff
@@ -186,7 +188,9 @@ private CustomAttributesBag<CSharpAttributeData> GetAttributesBag()
                 Debug.Assert(wasCompletedThisThread);
             }
 
+#nullable disable // Can '_lazyCustomAttributesBag' be null? https://github.com/dotnet/roslyn/issues/39166
```

Comment:
No. `LoadAndValidateAttributes` initializes the parameter always, except when `earlyDecodingOnly` is true. It is not in the call above.

Reply:
I'm planning to leave this one for a follow-up

Reply:
Updated